src/v2/Components/NavBar/NavBarLoggedInActions.tsx

```diff
@@ -0,0 +1,169 @@
+                new_notification_count: getNotificationCount(),
+                destination_path: "/works-for-you",
+              })
+            }}
+            {...anchorProps}
+          >
+            <NavBarItemUnfocusableAnchor href="/works-for-you" />
+
+            <BellIcon
+              title="Notifications"
+              // @ts-ignore
+              fill="currentColor"
+            />
+
+            {hasUnreadNotifications && (
+              <NavBarLoggedInActionsNotificationIndicator />
+            )}
+          </NavBarItemButton>
+        )}
+      </Dropdown>
+
+      <NavBarItemLink href="/user/conversations">
+        <EnvelopeIcon
+          title="Inbox"
+          // @ts-ignore
+          fill="currentColor"
+        />
+
+        {hasUnreadConversations && (
+          <NavBarLoggedInActionsNotificationIndicator />
+        )}
+      </NavBarItemLink>
+
+      <Dropdown dropdown={<NavBarUserMenu />} placement="bottom-end" offset={0}>
+        {({ anchorRef, anchorProps, visible }) => (
+          <NavBarItemButton
+            ref={anchorRef as any}
+            px={0}
+            pl={1}
+            active={visible}
+            {...anchorProps}
+          >
+            <SoloIcon
+              title="Your account"
+              // @ts-ignore
+              fill="currentColor"
+            />
+          </NavBarItemButton>
+        )}
+      </Dropdown>
+    </>
+  )
+}
+
+export const NavBarLoggedInActionsQueryRenderer: React.FC<{}> = () => {
+  const { relayEnvironment } = useContext(SystemContext)
+
+  return isServer ? (
+    <NavBarLoggedInActions />
+  ) : (
+    <SystemQueryRenderer<NavBarLoggedInActionsQuery>
+      environment={relayEnvironment}
+      query={graphql`
+        query NavBarLoggedInActionsQuery {
+          me {
+            unreadNotificationsCount
+            unreadConversationCount
+            followsAndSaves {
+              notifications: bundledArtworksByArtistConnection(
+                sort: PUBLISHED_AT_DESC
+                first: 10
+              ) @connection(key: "WorksForYou_notifications") {
+                edges {
+                  node {
+                    href
+                    summary
+                    artists
+                    published_at: publishedAt(format: "MMM DD")
+                    image {
+                      resized(height: 40, width: 40) {
+                        url
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      `}
+      render={({ props }) => {
+        return <NavBarLoggedInActions {...props} />
+      }}
+    />
+  )
+}
+
+export const NavBarLoggedInActionsNotificationIndicator = styled(Flex).attrs({
+  bg: "red100",
+})`
+  border-radius: 50%;
+  width: 6px;
+  height: 6px;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: ${themeGet("space.2")};
+  right: 5px;
+`
```

Comment:
This is also removed from immediately executing so logged in users should see one less graphql request and a few less image requests on page loads